wifi-connect-exclave/Dockerfile.template

```diff
@@ -1,41 +1,69 @@
 # Fin image contains → 1 container (Exclave Testing) →  Exclave (requires rust), node, NPM, codecept, balenaSDK, testbotSDK, balenaCLI, credentials for the CLI and NPM token in secrets
 
+FROM debian:buster
 
-#Ryan's past exclave Dockerfile
-FROM rust:1.31
+# build args (override with --build-arg NAME=VAL)
+ARG RUST_VERSION=1.31
+ARG NODE_VERSION=12.6.0
+ARG BALENA_CLI_VERSION=12.33.0
+ARG EXCLAVE_VERSION=v0.2.6
 
-RUN apt-get update 
-RUN apt-get install -y --no-install-recommends git ca-certificates wget
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+# set debconf to run non-interactively
+ENV DEBIAN_FRONTEND noninteractive
 
-ENV NODE_VERSION=12.6.0
-RUN apt install -y curl
-RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
-ENV NVM_DIR=/root/.nvm
-RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
-RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
-RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
-ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
-RUN node --version
-RUN npm --version
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        python \
+        git \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        git \
+        wget && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# set the SHELL option -o pipefail before RUN with a pipe in
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Would love to know what does this do?

Reply:
https://github.com/hadolint/hadolint/wiki/DL4006

Reply:
Life effectively improved. I used to do my lining here https://www.fromlatest.io/